tech/languages/csharp/csharp-ide.md

```diff
@@ -26,7 +26,9 @@ JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core
 
 _.NET Core and Mono_
 
-You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE. Relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
+* Eclipse will not work with [our .NET Core packages](/tech/languages/csharp/dotnet-installation.html) due to an [upstream issue](https://github.com/dotnet/source-build/issues/125). You can still use it for mono, or with [Microsoft's dotnet packages](https://www.microsoft.com/net/core#linuxfedora).
```

Comment:
Just one question:

`due to an [upstream issue](https://github.com/dotnet/source-build/issues/125)`
Is this really relevant info for a developer?

`It does not currently work...` is IMHO sufficient, but if you insist, I'll just merge it.

Reply:
The curious ones can take a look what's the issue, maybe contribute, who knows...

With my exhausted brain I totally forgot that it's a thing for VSCode as well so... I'm adding a note for that as well... It's ugly.